malindex.py

```diff
@@ -168,7 +168,7 @@ def fid2path(self, fid):
                 with open(self.pathlistfile,  "rb") as f:
                     for i, l in enumerate(f):
                         if self.numsamples is None or i < self.numsamples:
-                            self.paths.append(l.split(b" ")[0].decode().rstrip("\n"))
+                            self.paths.append(l.strip().decode())
```

Comment:
This does not respect whitespace trimming, right? Any space (also at the end of a file) will be part of the path.